nimsuggest/nimsuggest.nim

```diff
@@ -275,7 +279,7 @@ proc connectToNextFreePort(server: Socket, host: string): Port =
   result = port
 
 type
-  ThreadParams = tuple[port: Port; address: string]
+  ThreadParams = tuple[port: Port; address: string; findPort: bool]
```

Comment:
If you use a global already there is no point in passing it to the thread explicitly.